site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -9,54 +9,51 @@ messages to Eclipse Hono&trade;'s Telemetry, Event and Command & Control endpoin
 
 ## Device Authentication
 
-By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during connection establishment.
+By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during
+connection establishment.
 This is the preferred way for devices to publish data via protocol adapters. The AMQP adapter supports both the
-[SASL PLAIN](https://tools.ietf.org/html/rfc4616) and [SASL EXTERNAL](https://tools.ietf.org/html/rfc4422) authentication mechanisms.
-The former uses a *username* and *password* to authenticate to the adapter while the latter uses an X.509 client certificate.
+[SASL PLAIN](https://tools.ietf.org/html/rfc4616) and [SASL EXTERNAL](https://tools.ietf.org/html/rfc4422)
+authentication mechanisms.
 
-This guide provides examples for publishing telemetry and events for *authenticated* (using SASL PLAIN) and *unauthenticated* clients.
+This guide provides examples for publishing telemetry and events for *authenticated* (using SASL PLAIN) and
+*unauthenticated* clients.
 
 **NB** The AMQP adapter can be configured to *allow* unauthenticated devices to connect by setting configuration variable
-`HONO_AMQP_AUTHENTICATION_REQUIRED` to `false`.
+*HONO_AMQP_AUTHENTICATION_REQUIRED* to `false`.
 
 ### SASL PLAIN Authentication
 
-The AMQP adapter supports authenticating clients using a *username* and *password*. This means that clients need to provide a *username* and a *password*
-when connecting to the AMQP adapter. The *username* must match the pattern [*auth-id@tenant*], e.g. `sensor1@DEFAULT_TENANT`.
+The AMQP adapter supports authenticating clients using the SASL PLAIN mechanism. This means that clients need to
+provide a username and a password in the *authcid* and *passwd* fields of their SASL response as defined in
+[RFC 4616, Section 2](https://datatracker.ietf.org/doc/html/rfc4616#section-2) when connecting to the AMQP adapter.
+The username provided in the *authcid* field must match the pattern *auth-id@tenant*, e.g. `sensor1@DEFAULT_TENANT`.
 
-The adapter verifies the credentials provided by the client against the credentials that the
-[configured Credentials service]({{< relref "/admin-guide/common-config#credentials-service-connection-configuration" >}}) has on record for the client.
-If the credentials match, the client device can proceed to publish messages to Hono.
+The adapter then verifies the provided username and password against the credentials that the
+[configured Credentials service]({{< relref "/admin-guide/common-config#credentials-service-connection-configuration" >}})
+has on record for the client as described in
+[Username/Password based Authentication]({{< relref "/concepts/device-identity#username-password-based-authentication" >}}).
```

Comment:
Link anchor doesn't match - `relref` should be be `/concepts/device-identity#usernamepassword-based-authentication`.
This needs to be fixed in the corresponding chapters in the other protocol adapter guides as well.